python/dgl/transforms/module.py

```diff
@@ -96,6 +98,84 @@ def __call__(self, g):
     def __repr__(self):
         return self.__class__.__name__ + '()'
 
+class RWPE(BaseTransform):
+    r"""Random Walk Positional Encoding, as introduced in
+    `Graph Neural Networks with Learnable Structural and Positional Representations
+    <https://arxiv.org/abs/2110.07875>`__
+
+    This module only works for homogeneous graphs.
+    
+    Parameters
+    ----------
+    k : int
+        Number of random walk steps. The paper found the best value to be 16 and 20
+        for two experiments.
+    feat_name : str, optional
+        Name to store the computed positional encodings in ndata.
+
+    Example
+    -------
+
+    >>> import dgl
+    >>> from dgl import RWPE
+
+    >>> transform = RWPE(k=2)
+    >>> g = dgl.graph(([0, 1, 1], [1, 1, 0]))
+    >>> g = transform(g)
+    >>> print(g.ndata['PE'])
+    tensor([[0.0000, 0.5000],
+            [0.5000, 0.7500]])
+    """
+    def __init__(self, k, feat_name='PE'):
```

Comment:
We should have many `transforms` that saves data to `ndata` or `edata`. Do we have a naming convention here? @mufeili

Reply:
For now, we can start with having transforms for the same purpose to use the same name(s) like `PE` for the PE transforms.